deck/packages/core/src/pipeline/executions/executionGroup/ExecutionGroup.tsx

```diff
@@ -153,7 +154,10 @@ export class ExecutionGroup extends React.PureComponent<IExecutionGroupProps, IE
             .then((plan) => resolve(plan))
             .catch(() => resolve(config));
         } else {
-          resolve(config);
+          PipelineConfigService.getPipelinesForApplication(this.props.application.name).then((pipelines) => {
```

Comment:
This retrieves all pipeline configs in an application, yes?  Possible to change this to only retrieve the pipeline config with the specific id the code cares about here?  That'll reduce the load a ton on front50 for big applications.